locales/en/app.ftl

```diff
@@ -798,3 +798,21 @@ vpn-promo-cta = Get { -brand-mozilla-vpn }
 
 vpn-promo-headline-new = Save 50% with a full year subscription
 vpn-promo-copy-new = Protect your online data—and choose a VPN subscription plan that works for you.
+
+# VPN educational/promotional banner
+vpn-banner-location = Your location:
+vpn-banner-protect-yourself = Protect yourself
+vpn-banner-with-moz-vpn = with { -brand-mozilla-vpn }.
```

Comment:
Please don't split sentences, this should be a single string.

See also [this doc](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#make-strings-properly-localizable) (this is basically concatenation).

Reply:
Thanks for the helpful link!  Assuming no objections I would like to add the documentation to this repo's README.

I'm trying to follow an example suggested in your doc by using this string that includes HTML: 
```
vpn-banner-protect-yourself = <output class="vpn-banner-red">Protect yourself</output> with { -brand-mozilla-vpn }.
```

Unfortunately this does not work as the tags are printed literally.  There are no additional steps listed in your doc – is there another resource or example that I can look at?  Or does is that functionality not supported for our setup?

Reply:
Unfortunately I'm not a dev, so I can only help up to a certain point. I thought this project was using Fluent React, so you'd need [React Overlays](https://github.com/projectfluent/fluent.js/wiki/React-Overlays), but I can't find references in packages.json

I see strings with HTML markup in the file, so it's definitely possible. Maybe previous devs on the project have pointers?

Reply:
The way HTML is currently processed in this repo requires much additional code customized for each individual case.  I found a much simpler way using a different syntax in the templating engine.  This syntax `{{{ string }}}` requires no javascript to process the markup in the string.

Armed with this knowledge, I'll redo all strings as needed with your comments addressed.  Thanks for the help!